src/drivers/drv_sensor.h

```diff
@@ -165,6 +165,7 @@
 
 #define DRV_ADC_DEVTYPE_ADS1115	0x90
 #define DRV_DIST_DEVTYPE_VL53L1X 0x91
+#define DRV_DIST_DEVTYPE_GY_US42 0x9f
```

Comment:
```suggestion
#define DRV_DIST_DEVTYPE_GY_US42 0x92
```

Is there a reason it jumps forward? It doesn't really matter yet until we start really using device IDs for range finders, but I'd like to try and minimize gaps.